examples/panel/rag-news/rag_news.py

```diff
@@ -99,3 +99,4 @@ def compute_embeddings():
 
     # compute embeddings
     compute_embeddings()
+    
```

Comment:
Not a blank line exactyl.After copying the file from main just type 1 enter. that usually works for me

Reply:
tried both but it is still showing up